Training model=base
{'config_format': 'markov', 'batch_size': 16, 'acc_steps': 1, 'seed': 0, 'device': device(type='cpu'), 'iterations': 500, 'lr': 0.002, 'warmup_percent': 0.02, 'weight_decay': 0.001, 'beta1': 0.9, 'beta2': 0.95, 'scheduler': 'cos', 'opt': 'adamw', 'eval_freq': 1, 'results_base_folder': './exps', 'grad_clip': 0.0, 'dataset': 'markov', 'vocab_size': 2, 'data_in_ram': False, 'model': 'base', 'use_pretrained': 'none', 'dropout': 0, 'n_head': 1, 'n_layer': 1, 'n_embd': 4, 'sequence_length': 1024, 'dtype': torch.float16, 'bias': False, 'no_compile': False, 'wandb': True, 'wandb_project': 'test', 'wandb_run_prefix': 'none', 'eval_seq_prefix': '0', 'distributed_backend': None, 'p': 0.2, 'q': 0.3, 'memory': -1, 'initial': 'steady', 'no_tying': False, 'init': 'base', 'v_std': 0, 'world_size': 1}
4293
Compiling model ...
tensor([0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0,
        0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0,
        0, 0, 0, 0, 0, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
        0, 0, 0, 1, 0, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1,
        1, 1, 1, 0])
wte:
Parameter containing:
tensor([[-0.0147],
        [-0.0075],
        [ 0.0319],
        [-0.0341]], requires_grad=True)
wpe:
Parameter containing:
tensor([[-0.0014, -0.0157, -0.0177, -0.0299],
        [-0.0054,  0.0292, -0.0158,  0.0195],
        [-0.0129,  0.0288,  0.0016, -0.0243],
        ...,
        [-0.0018,  0.0048, -0.0366,  0.0299],
        [ 0.0048, -0.0277,  0.0225,  0.0029],
        [ 0.0129,  0.0060, -0.0385,  0.0009]], requires_grad=True)
W_Q W_K W_V:
[[-5.62977418e-03  8.01561400e-05  1.86011624e-02  1.31880995e-02]
 [ 3.30119394e-02 -1.85932610e-02 -8.31654202e-03 -2.05520801e-02]
 [-4.87800222e-04 -9.21675283e-03  2.05956046e-02  1.39029967e-02]
 [-6.20508613e-03  3.77451838e-03 -5.69576351e-03 -1.10556604e-02]
 [-8.98501929e-03  1.19461687e-02 -1.22771990e-02 -1.32652633e-02]
 [-1.58547591e-02  8.57592933e-03 -3.58993351e-03 -1.56347621e-02]
 [ 4.36439551e-02 -2.90442258e-03 -1.58269908e-02 -6.16173446e-03]
 [-6.27195986e-04  2.83308588e-02  5.53012602e-02 -2.19565872e-02]
 [-3.30575332e-02 -1.07474031e-03 -2.37902906e-02 -1.59071665e-02]
 [-1.67845022e-02 -1.54180527e-02  2.47932412e-02  6.13041874e-03]
 [-1.67541485e-02  4.03625937e-03  4.19070907e-02 -2.24560276e-02]
 [ 1.15523422e-02  1.21771051e-02  3.47881019e-03 -6.65960647e-03]]
Approximation error:
tensor(0.0005, grad_fn=<CompiledFunctionBackward>)
c_fc:
Parameter containing:
tensor([[ 0.0028, -0.0186, -0.0147,  0.0129],
        [ 0.0352, -0.0021, -0.0012, -0.0149],
        [ 0.0314, -0.0290, -0.0199, -0.0090],
        [ 0.0243, -0.0127,  0.0034, -0.0182],
        [ 0.0213,  0.0059, -0.0394, -0.0200],
        [ 0.0056, -0.0056,  0.0075, -0.0075],
        [ 0.0015,  0.0063,  0.0229,  0.0474],
        [ 0.0219, -0.0102,  0.0220, -0.0149],
        [-0.0206, -0.0033, -0.0046,  0.0358],
        [ 0.0045,  0.0540,  0.0014,  0.0224],
        [-0.0208,  0.0135,  0.0122,  0.0162],
        [-0.0245,  0.0091,  0.0116, -0.0331],
        [ 0.0205, -0.0143, -0.0068,  0.0167],
        [-0.0029,  0.0046, -0.0177, -0.0176],
        [ 0.0030,  0.0094, -0.0187, -0.0189],
        [ 0.0121,  0.0085,  0.0109,  0.0098]], requires_grad=True)
c_proj:
Parameter containing:
tensor([[ 0.0079, -0.0088,  0.0169,  0.0015, -0.0118, -0.0175, -0.0102, -0.0037,
         -0.0010,  0.0056, -0.0100, -0.0073,  0.0024, -0.0319, -0.0055, -0.0044],
        [ 0.0272, -0.0124,  0.0176, -0.0039, -0.0010, -0.0013,  0.0148, -0.0187,
         -0.0074,  0.0045,  0.0033,  0.0127,  0.0040, -0.0154,  0.0031, -0.0005],
        [-0.0053, -0.0167,  0.0040, -0.0220,  0.0120, -0.0027, -0.0112,  0.0019,
         -0.0119,  0.0054, -0.0273, -0.0357,  0.0184,  0.0090, -0.0118,  0.0057],
        [ 0.0006,  0.0170,  0.0032,  0.0077,  0.0201, -0.0136, -0.0189, -0.0004,
         -0.0388, -0.0092, -0.0021,  0.0119,  0.0067,  0.0058, -0.0240, -0.0252]],
       requires_grad=True)
Traceback (most recent call last):
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builder.py", line 1126, in wrap_numpy_ndarray
    value.flags.writeable = True
    ^^^^^^^^^^^^^^^^^^^^^
ValueError: cannot set WRITEABLE flag to True of this array
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/main.py", line 153, in <module>
    main(args)
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/main.py", line 135, in main
    stats = train(model, opt, P, scheduler, args.iterations, args.acc_steps, args.batch_size, args.sequence_length, generator,
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/optim/base.py", line 32, in train_base
    outputs = model(x, targets=y)
              ^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/eval_frame.py", line 451, in _fn
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 282, in forward
    def forward(self, idx, targets=None, get_logits=False):
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 286, in torch_dynamo_resume_in_forward_at_286
    print(idx[0,:100])
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 294, in torch_dynamo_resume_in_forward_at_294
    print("wte:")
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 295, in torch_dynamo_resume_in_forward_at_295
    print(self.transformer.wte.weight)
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 296, in torch_dynamo_resume_in_forward_at_296
    print("wpe:")
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 316, in torch_dynamo_resume_in_forward_at_297
    x = block(x)
        ^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 170, in forward
    def forward(self, x):
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 172, in torch_dynamo_resume_in_forward_at_172
    y = self.attn(x)
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 174, in torch_dynamo_resume_in_forward_at_174
    print("Approximation error:")
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 175, in torch_dynamo_resume_in_forward_at_175
    print(torch.mean(torch.norm(y[0], dim=1) / torch.norm(z[0], dim=1)))
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1532, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1541, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 131, in forward
    def forward(self, x):
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 137, in torch_dynamo_resume_in_forward_at_137
    print("c_fc:")
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 138, in torch_dynamo_resume_in_forward_at_138
    print(self.c_fc.weight)
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 139, in torch_dynamo_resume_in_forward_at_139
    print("c_proj:")
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 140, in torch_dynamo_resume_in_forward_at_140
    print(self.c_proj.weight)
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 142, in torch_dynamo_resume_in_forward_at_142
    plt.ioff()
  File "/Users/bondasch/Downloads/IT/Markov/Markov-Simple/src/models/base.py", line 143, in torch_dynamo_resume_in_forward_at_143
    img_c_fc = plt.matshow(self.c_fc.weight.numpy(force=True))
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 449, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 400, in wrapper_fn
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 390, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1877, in log
    self._log(data=data, step=step, commit=commit)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1641, in _log
    self._partial_history_callback(data, step, commit)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/wandb_run.py", line 1513, in _partial_history_callback
    self._backend.interface.publish_partial_history(
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/interface/interface.py", line 600, in publish_partial_history
    data = history_dict_to_json(run, data, step=user_step, ignore_copy_err=True)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/data_types/utils.py", line 52, in history_dict_to_json
    payload[key] = val_to_json(
                   ^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/data_types/utils.py", line 86, in val_to_json
    val = Plotly.make_plot_media(val)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/data_types/plotly.py", line 48, in make_plot_media
    return Image(val)
           ^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/data_types/image.py", line 179, in __init__
    self._initialize_from_data(data_or_path, mode, file_type)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/wandb/sdk/data_types/image.py", line 302, in _initialize_from_data
    util.ensure_matplotlib_figure(data).savefig(buf, format="png")
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/figure.py", line 3390, in savefig
    self.canvas.print_figure(fname, **kwargs)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backend_bases.py", line 2187, in print_figure
    result = print_method(
             ^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backend_bases.py", line 2043, in <lambda>
    print_method = functools.wraps(meth)(lambda *args, **kwargs: meth(
                                                                 ^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 497, in print_png
    self._print_pil(filename_or_obj, "png", pil_kwargs, metadata)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 445, in _print_pil
    FigureCanvasAgg.draw(self)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 388, in draw
    self.figure.draw(self.renderer)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/artist.py", line 95, in draw_wrapper
    result = draw(artist, renderer, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/artist.py", line 72, in draw_wrapper
    return draw(artist, renderer)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/figure.py", line 3153, in draw
    self.patch.draw(renderer)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/artist.py", line 72, in draw_wrapper
    return draw(artist, renderer)
           ^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/patches.py", line 588, in draw
    self._draw_paths_with_artist_properties(
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/patches.py", line 573, in _draw_paths_with_artist_properties
    renderer.draw_path(gc, *draw_path_args)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 97, in draw_path
    nmax = mpl.rcParams['agg.path.chunksize']  # here at least for testing
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 921, in catch_errors
    return callback(frame, cache_entry, hooks, frame_state, skip=1)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 786, in _convert_frame
    result = inner_convert(
             ^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 400, in _convert_frame_assert
    return _compile(
           ^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/contextlib.py", line 81, in inner
    return func(*args, **kwds)
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 676, in _compile
    guarded_code = compile_inner(code, one_graph, hooks, transform)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/utils.py", line 262, in time_wrapper
    r = func(*args, **kwargs)
        ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 535, in compile_inner
    out_code = transform_code_object(code, transform)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/bytecode_transformation.py", line 1036, in transform_code_object
    transformations(instructions, code_options)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 165, in _fn
    return fn(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/convert_frame.py", line 500, in transform
    tracer.run()
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 2149, in run
    super().run()
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 810, in run
    and self.step()
        ^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 773, in step
    getattr(self, inst.opname)(inst)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 1311, in LOAD_ATTR
    result = BuiltinVariable(getattr).call_function(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builtin.py", line 687, in call_function
    result = handler(tx, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builtin.py", line 1277, in call_getattr
    return obj.var_getattr(tx, name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/user_defined.py", line 746, in var_getattr
    ).call_function(tx, [], {})
      ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/functions.py", line 335, in call_function
    return super().call_function(tx, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/functions.py", line 289, in call_function
    return super().call_function(tx, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/functions.py", line 90, in call_function
    return tx.inline_user_function_return(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 680, in inline_user_function_return
    return InliningInstructionTranslator.inline_call(self, fn, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 2285, in inline_call
    return cls.inline_call_(parent, func, args, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 2399, in inline_call_
    tracer.run()
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 810, in run
    and self.step()
        ^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 773, in step
    getattr(self, inst.opname)(inst)
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/symbolic_convert.py", line 1311, in LOAD_ATTR
    result = BuiltinVariable(getattr).call_function(
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builtin.py", line 687, in call_function
    result = handler(tx, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builtin.py", line 1277, in call_getattr
    return obj.var_getattr(tx, name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/user_defined.py", line 808, in var_getattr
    return VariableBuilder(tx, source)(subobj)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builder.py", line 269, in __call__
    vt = self._wrap(value)
         ^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builder.py", line 402, in _wrap
    return type_dispatch(self, value)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/torch/_dynamo/variables/builder.py", line 1130, in wrap_numpy_ndarray
    assert isinstance(value.base, np.nditer)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
AssertionError:
from user code:
   File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/backends/backend_agg.py", line 98, in torch_dynamo_resume_in_draw_path_at_97
    npts = path.vertices.shape[0]
  File "/Users/bondasch/anaconda3/lib/python3.11/site-packages/matplotlib/path.py", line 214, in vertices
    return self._vertices
Set TORCH_LOGS="+dynamo" and TORCHDYNAMO_VERBOSE=1 for more information
You can suppress this exception and fall back to eager by setting:
    import torch._dynamo
    torch._dynamo.config.suppress_errors = True